examples/sample-app/README.md

```diff
@@ -462,10 +462,11 @@ Another interesting example is deleting a pod.
 
 Cleaning Up
 -----------
-To clean up all of your environment, you can run the script:
+To empty your project, deleting all BuildConfigs, ImageStreams, DeploymentConfigs,
+Services, etc, you can use:
 
-        $ sudo ./cleanup.sh
+    $ oc delete all --all
```

Comment:
I don't want us to have cleanup scripts.  We have running in a container
for that, as well as other tools.  Since we can now effectively purge the
etcd and config dirs when running in a container, it would be more
appropriate to document *what* we create so that users can make their own
choice.  Cleanup just isn't very relevant to most people trying us out.

On Jul 3, 2015, at 11:55 AM, Ben Parees <notifications@github.com> wrote:

In examples/sample-app/README.md
<https://github.com/openshift/origin/pull/3564#discussion_r33872067>:

>
> -        $ sudo ./cleanup.sh
> +    $ oc delete all --all

this is not the equivalent to what cleanup was doing.

1) you're not stopping the openshift process
2) you need to run this as cluster-admin if you want it to really work and
these instructions don't make that clear.

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/3564/files#r33872067>.